plugins/generic/googleAnalytics/locale/sv_SE/locale.xml

```diff
@@ -10,7 +10,7 @@
   * Copyright (c) 2003-2016 John Willinsky
   * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
   *
-  * Google Analytics plugin localization strings for the en_US (U.S. English) locale.
+  * Google Analytics plugin localization strings for the sv_SE (Svenska) locale.
```

Comment:
Just FYI: In general I've been removing these lines referring to the language of the file -- it's a lot of pain to maintain them and there's no real benefit.